cli/src/commands/service/connect.ts

```diff
@@ -29,6 +29,11 @@ export default class Spawn extends Command {
       char: 't',
       description: 'An optional image tag or image digest to use when spawning a backend.',
     }),
+    cluster: Flags.string({
+      char: 'c',
+      description:
+        'The cluster to to spawn the backend in (only relevant if you are running multiple clusters with Jamsocket).',
```

Comment:
repeated to's

Reply:
ah good catch! thanks!

Reply:
https://github.com/jamsocket/jamsocket/pull/145